extension/src/experiments/columns/collect/deps.ts

```diff
@@ -37,7 +37,7 @@ export const collectDeps = (acc: ColumnAccumulator, data: ExperimentFields) => {
       path,
       buildDepPath(...limitedDepthAncestors),
       [ColumnType.DEPS, file],
-      label,
+      limitedDepthAncestors.length === 0 ? file : label,
```

Comment:
[Q] How does this work for 1/2? I don't understand what is happening here.

Reply:
When it comes a config of `2`, all of the ancestors are joined in `limitedDepthAncestors` and the value column label stays as is. 
But when it comes to a config of `1`, there are no `limitedDepthAncestors` since there is only one cell so `limitAncestorDepth` returns an empty array and the value column `label` needs to hold all ancestors plus the label aka `file`. 

Hopefully, that answers your question! Maybe there is a better way to go about this though 🤔

Reply:
I would pull this inline value out, give it an explanatory name and probably even access the file config value instead of doing this here....

After reading further:

All of the `limitedDepthAncestors.length === 0 ?` ternary logic should live together in the same place / function. Can it be consolidated somehow? It feels like the `limitAncestorDepth` function should return both the `label` and `limitedDepthAncestors` if possible.  That would mean that all of this logic is encapsulated and we don't have to repeat the transformations in a few places.

Reply:
Makes sense, thanks! Moved label creation to limitAncestorDepth.